libavcodec/vvc/vvcdsp.h

```diff
@@ -111,11 +112,10 @@ typedef struct VVCIntraDSPContext {
 } VVCIntraDSPContext;
 
 typedef struct VVCItxDSPContext {
-    void (*add_residual)(uint8_t *dst, const int *res, int width, int height, ptrdiff_t stride);
-    void (*add_residual_joint)(uint8_t *dst, const int *res, int width, int height, ptrdiff_t stride, int c_sign, int shift);
-    void (*pred_residual_joint)(int *buf, int width, int height, int c_sign, int shift);
-
-    void (*itx[N_TX_TYPE][N_TX_SIZE])(int *out, ptrdiff_t out_step, const int *in, ptrdiff_t in_step);
+    void (*add_residual)(uint8_t *dst, const int16_t *res, int width, int height, ptrdiff_t stride);
+    void (*add_residual_joint)(uint8_t *dst, const int16_t *res, int width, int height, ptrdiff_t stride, int c_sign, int shift);
+    void (*pred_residual_joint)(int16_t *buf, int width, int height, int c_sign, int shift);
+    void (*itx[N_TX_TYPE][N_TX_TYPE][N_TX_SIZE][N_TX_SIZE])(int16_t *dst, const int *coeff, int nzw, int log2_transform_range);
```

Comment:
DST7 64 and DST7 2 is not existed. 
this will make the function include a a NULL pointer. 

Could we define like this:
```
Enum TxType {
   DCT2_1,
   DCT2_2,
   ....
   DCT2_64,
   DST7_1
   DST7_4
   ...
   DST7_32
};
```

then we can use define itx as itx[TxType][TxType]

Reply:
Implemented in 140554f086901a0a898ea3daefb16e7c37e5ccee.